### DISCUSSION TITLE
Github Actions issue with templated workflow run for Azure Policy deployment

### DISCUSSION THREAD

**Discussion by qmalik111** [community/community#75746] [Actions]
### Select Topic Area

Bug

### Body

We have a .JSON file as a template that is being used to deploy policies in an Azure tenant environment. However, when we are running this workflow. We are finding an issue with the policy definition ID. It seems to be picking up a ‘/’ at the end which is causing an issue as the path becomes lost. I can insert a screenshot if required, we have since resolved the issue by deploying it manually through the Azure portal, which defeats the point as we are carrying out the pipeline work through automation.

However, can this bug please be investigated.

**Comment by AidenJamesYt** [comment]
It appears that there's an issue with the policy definition ID in the JSON template that's causing the deployment to fail. The additional '/' at the end of the ID is likely causing the path to be incorrect, preventing the deployment from finding the correct policy definition.

To address this issue, you can try the following steps:

Review the JSON template: Carefully inspect the JSON template to ensure that the policy definition ID is accurate and doesn't contain any extraneous characters like the trailing '/'.

Validate the policy definition ID: Verify that the policy definition ID exists in the Azure tenant environment. You can do this by manually checking the Azure Policy definitions list or using tools like the Azure CLI.

Update the JSON template: If the policy definition ID is correct and the issue persists, try removing the trailing '/' from the ID in the JSON template. This should resolve the path issue and allow the deployment to proceed.

Test the updated template: Once you've updated the JSON template, save the changes and re-run the workflow. This should hopefully allow the deployment to complete successfully without any errors related to the policy definition ID.
